openslides/core/static/templates/core/customslide-list.html

```diff
@@ -48,15 +48,97 @@ <h1 translate>Custom slides</h1>
+        <a os-perms="core.can_manage_projector"
+            class="btn btn-default btn-sm disabled"
+            title="{{ 'Zoom in' | translate}}">
+          <i class="fa fa-search-plus"></i>
+        </a>
+        <a os-perms="core.can_manage_projector"
+            class="btn btn-default btn-sm disabled"
+            title="{{ 'Zoom out' | translate}}">
+          <i class="fa fa-search-minus"></i>
+        </a>
+        <a os-perms="core.can_manage_projector"
+            class="btn btn-default btn-sm disabled"
+            title="{{ 'Scroll visible view up' | translate}}">
+          <i class="fa fa-arrow-up"></i>
+        </a>
+        <a os-perms="core.can_manage_projector"
+            class="btn btn-default btn-sm disabled"
+            title="{{ 'Scroll visible view down' | translate}}">
+          <i class="fa fa-arrow-down"></i>
+        </a>
+      </div>
+
+      <!-- countdowns -->
+      <div os-perms-lite="core.can_manage_projector">
+        <h3 translate>Countdown</h3>
+        <div ng-repeat="countdown in countdowns" class="countdown panel panel-default">
+          <div class="panel-heading">
+            Countdown {{ $index + 1 }}
+            <!-- remove countdown button -->
+            <button type="button" class="close"
+                ng-click="removeCountdown(countdown)"
+                title="{{ 'Remove countdown' | translate}}">
+                &times;
+            </button>
+          </div>
+          <div class="panel-body">
+            <!-- project countdown button -->
+            <a class="btn btn-default btn-sm"
+                ng-model="countdown.visible"
+                ng-class="{ 'btn-primary': countdown.visible }"
+                title="{{ 'Project countdown' | translate }}">
+              <i class="fa fa-video-camera"></i>
+            </a>
+            &nbsp;&nbsp;
+            <!-- countdown controls -->
+            <a class="btn btn-default vcenter"
+                ng-click="stopCountdown(countdown)"
+                title="{{ 'Reset countdown' | translate}}">
+              <i class="fa fa-stop"></i>
+            </a>
+            <a class="btn btn-default vcenter"
+                ng-click="startCountdown(countdown)"
+                title="{{ 'Start countdown' | translate}}">
+              <i class="fa fa-play"></i>
+            </a>
+            <span class="countdown_timer vcenter">{{ countdown.countdown_time }}</span>
+          </div>
+        </div>
+        <!-- Add countdown button -->
+        <a ng-click="addCountdown()"
+            class="btn btn-default btn-sm"
+            title="{{ 'Add countdown' | translate}}">
+          <i class="fa fa-plus"></i> <translate>Add new countdown</translate>
+        </a>
+      </div>
+
+       <!-- messages -->
+      <div os-perms-lite="core.can_manage_projector">
+        <h3 translate>Messages</h3>
+        <ul ng-repeat="message in messages" class="message list-group">
+          <li class="list-group-item">{{message}}</li>
```

Comment:
@normanjaeckel I can't test the projector messages because the rest framework view shows always "Authentication credentials were not provided.".